README.md

```diff
@@ -102,6 +102,25 @@ the secrets stored in temp files and in the Python process environment are gone.
 
     This flag can be useful for when you have secrets that you don't need access to for development. For example API keys for monitoring tools. This flag can be used multiple times.
 
+* `-e, --environment` Specify section (environment) to parse from secret YAML
+
+    This flag specifies which specific environment/section to parse from the secrets YAML file (or string). In addition, it will also enable the usage of a `common` section which will be inherited by other sections/environments. In other words, if your `secrets.yaml` looks something like this:
+
+```yaml
+common:
```

Comment:
I think the `-D` feature is good and useful, but I feel that being able to have a complete view of all environment vars that each environment uses is easier to digest; if anything, it also seems to me that there's less guess work and/or less potential for unexpected behavior (ie. "I thought this env was defined in the bucket").

Besides that, I think one should always make new commits (or leave a visible) trace when modifying an applications environment.